core/lib/types/EilenbergMacLane1.agda

```diff
@@ -31,12 +34,12 @@ module _ {G : Group i} where
   embase = embase' G
   emloop = emloop' G
   emloop-comp = emloop-comp' G
+  emloop-coh = emloop-coh' G
 
   instance
-    EM₁-level : {n : ℕ₋₂} → has-level (S (S (S n))) (EM₁ G)
+    EM₁-level : {n : ℕ₋₂} → has-level (S (S (S (S n)))) (EM₁ G)
```

Comment:
The current convention would have `EM₁-level : has-level 2 (EM₁ G)`. If you want to omit `raise-level`, I wonder if it helps to make `raise-level` an instance? @guillaumebrunerie I wonder if you have considered making `raise-level` an instance before?